zebra-rpc/src/methods.rs

```diff
@@ -103,6 +105,22 @@ pub trait Rpc {
     /// zcashd reference: [`getrawmempool`](https://zcash.github.io/rpc/getrawmempool.html)
     #[rpc(name = "getrawmempool")]
     fn get_raw_mempool(&self) -> BoxFuture<Result<Vec<String>>>;
+
+    /// Returns the raw transaction data, as a [`GetRawTransaction`] JSON string or structure.
+    ///
+    /// zcashd reference: [`getrawtransaction`](https://zcash.github.io/rpc/getrawtransaction.html)
+    ///
+    /// # Parameters
+    ///
+    /// - `txid`: (string, required) The transaction ID of the transaction to be returned.
+    /// - `verbose`: (numeric, optional, default=0) If 0, return a string of hex-encoded data, otherwise return a JSON object.
+    /// - `blockhash`: (string, optional) The block in which to look for the transaction.
+    #[rpc(name = "getrawtransaction")]
+    fn get_raw_transaction(
+        &self,
+        txid: String,
```

Comment:
```suggestion
        txid_hex: String,
```

I think we don't need to match the name of the arguments from the zcashd RFC so we can call it `txid_hex` here and in the implementation so we get consistency with https://github.com/ZcashFoundation/zebra/blob/main/zebra-rpc/src/methods.rs#L72

Reply:
Done in bada4e3